HISTORY.md

```diff
@@ -1,3 +1,7 @@
+### ensmallen 1.14.2
+###### ????-??-??
+  * Minor documentation fixes (#95).
```

Comment:
Not really related, but what about adding https://github.com/mlpack/ensmallen/pull/92 to the history as well.

Reply:
Sure :)